packages/app/src/runner/aut-iframe.ts

```diff
@@ -82,6 +82,40 @@ export class AutIframe {
     return Cypress.cy.detachDom(this._contents())
   }
 
+  /**
+   * If the AUT is cross origin relative to top, a security error is thrown and the method returns false
+   * If the AUT is cross origin relative to top and chromeWebSecurity is false, origins of the AUT and top need to be compared and returns false
+   * Otherwise, if top and the AUT match origins, the method returns true.
+   * If the AUT origin is "about://blank", that means the src attribute has been stripped off the iframe and is adhering to same origin policy
+   */
+  doesAUTMatchTopOriginPolicy = () => {
```

Comment:
original in [runner-shared](https://github.com/cypress-io/cypress/pull/18075/files#diff-e79a069c075dafc43ba805a69ef368962f7fc50f54a07476b039dd0c470f04c0R78) in `9.x`